sonar-scanner-engine/src/main/java/org/sonar/scanner/repository/MetricsRepositoryLoader.java

```diff
@@ -19,8 +19,6 @@
  */
 package org.sonar.scanner.repository;
 
-import org.sonar.scanner.protocol.input.GlobalRepositories;
-
-public interface GlobalRepositoriesLoader {
-  GlobalRepositories load();
+public interface MetricsRepositoryLoader {
```

Comment:
![CRITICAL](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-critical.png) Annotate the "MetricsRepositoryLoader" interface with the @FunctionalInterface annotation [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3AS1609)